HPC/README.md

```diff
@@ -492,3 +492,17 @@ GCC	| 11	| -O3 -mcpu=neoverse-v1	| -fopenmp	| -ffast-math
 CLang/LLVM |	14 | -O3 -mcpu=neoverse-512tvb	| -fopenmp	| -ffast-math
 Arm Compiler for Linux |	23.04 |	-O3 -mcpu=neoverse-512tvb |	-fopenmp |	-ffast-math
 Nvidia HPC SDK |	23.1	| -O3 -tp=neoverse-v1	| -mp |	-fast
+
+### Common HPC Applications on Graviton
+Below is a list of some common HPC applications that run on Graviton.
+ISV                   | Application   | Release of support  | Additional Notes
+----------------------|---------------|---------------------|-----------------
+Ansys                 | Fluent        | v221                | [Graviton Applications (AWS)](https://aws.amazon.com/blogs/hpc/application-deep-dive-into-the-graviton3e-based-amazon-ec2-hpc7g-instance/)
+Ansys                 | LS-Dyna       | 12.1                | [Graviton Applications (AWS)](https://aws.amazon.com/blogs/hpc/application-deep-dive-into-the-graviton3e-based-amazon-ec2-hpc7g-instance/)
```

Comment:
we can also refer to this link 
https://rescale.com/blog/rescale-automates-the-deployment-of-ansys-ls-dyna-and-ansys-fluent-workloads-on-amazon-ec2-hpc7g-instances/

Reply:
Added.  Thank you.